src/main/java/com/st/utopia/cancellationservice/controller/CancellationServiceController.java

```diff
@@ -0,0 +1,51 @@
+package com.st.utopia.cancellationservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.st.utopia.cancellationservice.model.Ticket;
+import com.st.utopia.cancellationservice.service.TicketService;
+
+import javassist.NotFoundException;
+
+
+
+@RestController
+@RequestMapping("/Cancel")
+public class CancellationServiceController {
+	
+	@Autowired
+	private TicketService ticketservice;
+	
+	
+	@GetMapping("/FindTicket/{flightNumber}/{rowNumber}/{seat}")

```

Comment:
In my opinion it's generally a red flag when a URL is expected to include a capital letter, and especially if it uses "CamelCase" rather than using underscores (`_`) or hyphens (`-`) to separate words.